### PR TITLE
Adds get_forecast_attribute service to Weather

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

This adds the get_forecast_attribute service to `Weather`

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/100928
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/weather.markdown [hunk 1]
```diff
@@ -193,3 +193,99 @@ weather.toronto_forecast:
 ```
 
 {% enddetails %}
+
+## Service `weather.get_forecast_attribute`
+
+This service populates [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data)
+with a mapping of weather services and their associated forecasts.
```
Comment (emontnemery) [inline source/_integrations/weather.markdown:200]: This is just copied from the `weather.get_forecast` service, without adjusting the language to what the new service does.

File: source/_integrations/weather.markdown [hunk 2]
```diff
@@ -193,3 +193,99 @@ weather.toronto_forecast:
...
+| `humidity` | The relative humidity in %. | 82
+| `precipitation_probability` | The probability of precipitation in %. | 0
+| `precipitation` | The precipitation amount in the unit indicated by the `precipitation_unit` state attribute. | 0
+| `pressure` | The air pressure in the unit indicated by the `pressure_unit` state attribute. | 1019
+| `temperature` | The temperature in the unit indicated by the `temperature_unit` state attribute. If `templow` is also provided, this is the higher temperature. | 14.2
+| `templow` | The lower temperature in the unit indicated by the `temperature_unit` state attribute. | 5.0
+| `uv_index` | The UV index. | 3
+| `wind_bearing` | The wind bearing in azimuth angle (degrees) or 1-3 letter cardinal direction. | 268
+| `wind_gust_speed` | The wind gust speed in the unit indicated by the `wind_speed_unit` state attribute. | 34.41
+| `wind_speed` | The wind speed in the unit indicated by the `wind_speed_unit` state attribute. | 24.41
```
Comment (emontnemery) [inline source/_integrations/weather.markdown:240]: Again, this is all just copied, and doesn't seem to make sense

**Review by emontnemery — CHANGES_REQUESTED** [review]
The documentation does not match the new service to my understanding

**Comment by gjohansson-ST** [comment]
Closing as parent PR has been closed.

**Pull request closed without merging**